src/app/app.routing.module.ts

```diff
@@ -62,6 +62,10 @@ const routes: Routes = [
         path: 'admin/logging',
         loadChildren: './modules/logging-management/logging-management.module#LoggingManagementModule'
     },
+    {
+      path: 'admin/dashboard',
+      loadChildren: () => import('./modules/dashboard/dashboard.module').then(m => m.DashboardModule)
+    },
```

Comment:
Cosmetic: indentation is not aligned - we use 4 spaces.

```suggestion
    {
        path: 'admin/dashboard',
        loadChildren: () => import('./modules/dashboard/dashboard.module').then(m => m.DashboardModule)
    },
```

Reply:
Fixed :)